### DISCUSSION TITLE
How do I configure waybar to only display on one monitor?

### DISCUSSION THREAD

**Discussion by Miuler** [prasanthrangan/hyprdots#731] [Q&A]
I have two monitor, how do I configure waybar to only display on one monitor?

**Comment by kRHYME7** [comment]
in my system adding like this ``` "output": "eDP-1", ``` works. 
Edit your ``` ~/.config/waybar/modules/header.jsonc ```

should look like this
```
//   --// waybar config generated by wbarconfgen.sh //--   //

{
// sourced from header module //
    "output": "eDP-1",
    "layer": "top",
    "position": "${w_position}",
    "mod": "dock",
    "height": ${w_height},
    "exclusive": true,
    "passthrough": false,
    "gtk-layer-shell": true,

```
use ``` hyprctl monitors ``` to see what is your monitor name 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/8472f092-0603-41a7-8d2c-c59e86a84374)
